Packages/manifest.json

```diff
@@ -4,9 +4,9 @@
     "com.unity.ide.rider": "2.0.7",
     "com.unity.ide.visualstudio": "2.0.5",
     "com.unity.ide.vscode": "1.2.3",
-    "com.unity.test-framework": "1.1.20",
-    "com.unity.textmeshpro": "3.0.1",
-    "com.unity.timeline": "1.2.17",
+    "com.unity.test-framework": "1.1.22",
```

Comment:
we can probably remove the timeline and test framework, since we won't be using those anyway. but nbd. it's up to you.

Reply:
I could see timeline coming in handy cause we're doing stuff with cinemachine and potentially will want to throw some audio in alongside the cinemachine transitions, but yeah updating the packages was more just habit for starting new projects haha, I'd be fine with getting rid of the test framework or collaborate

Reply:
yeah test and collaborate can be removed